build.gradle.kts

```diff
@@ -15,6 +15,7 @@ repositories {
 dependencies {
     testImplementation(kotlin("test"))
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:$kotlinx_coroutines")
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-debug:$kotlinx_coroutines")
```

Comment:
should it be testImplementation?

Reply:
It shouldn't be at all :) I used it only for debugigng